ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/archive-search.component.scss

```diff
@@ -200,4 +200,94 @@ color: var(--vitamui-primary);
 padding-right: 10px;
 padding-left: 10px;
 font-size: 22px;
+}
+
+
+.search-criterias{
+    font-size: large;
+    font-weight: bold;
+    margin-top: 50px;
+}
+
+.no-results-center {
+    text-align: center;
+}
+
+.left-margin{
+    margin-left: 30%;
+}
+.show-panel {
+    display: none;
+}
+.hide-panel {
+    display: block;
+}
+
+::ng-deep .my-menu {
+    max-width: 380px;
+}
+
+.circle {
+
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 13px;
+position: static;
+width: 50px;
+height: 50px;
+left: 0px;
+top: 0px;
+
+/* Secondary/Secondary-500 */
+
+background: var(--vitamui-secondary);
+border-radius: 50px;
+
+/* Inside Auto Layout */
+
+flex: none;
+order: 0;
+flex-grow: 0;
+margin: 0px 20px;
+border: none;
+}
+
+.circle:focus{
+    border:none;
+    outline: -webkit-focus-ring-color auto 0px;
+    outline-color: -webkit-focus-ring-color;
+    outline-style: auto;
+    outline-width: 0px;
+}
+
+.primary-save-icon {
+    color: white;
+}
+
+.mat-boutton {
+    border: none;
```

Comment:
Je ne sais pas si c'est voulu, mais tous les boutons ou autres éléments simples ont été homogénéisé dans le cadre de la refonte vitamui. Ils sont disponible dans le starter-kit au besoin. Il faudrait peut-être s'aligner dessus ?

Reply:
c'est voulu cet override du boutton du menu contextuel